iFixit/react-commerce
# Refactor collections product grid

closes #1432 

## QA

We just need to verify that product list grid view looks the same

1. Open a [product list preview](https://react-commerce-git-refactor-collections-product-grid-ifixit.vercel.app/Parts)
2. Verify that the grid looks the same as production

    <img width="980" alt="Screenshot 2023-03-07 at 17 38 16" src="https://user-images.githubusercontent.com/4640135/223488474-930c7fce-2503-4622-9f5c-58b2b6bf3d19.png">
